docs/UserGuide.adoc

```diff
@@ -355,15 +373,15 @@ e.g. `/order add f/Roti Prata n/James Ho p/22224444 a/block 123, Clementi Rd, 12
 * *Listing orders* : `/order list`
 * *Edit order* : `/order edit INDEX [f/FOOD] [n/NAME] [p/PHONE_NUMBER] [a/ADDRESS] [dt/DATETIME]` +
 e.g. `/order edit 2 n/James Lee`
-* *Find order* : `/order find [n/NAME] [p/PHONE]` +
+* *Find order* : `/order find [n/NAME] [p/PHONE] [a/ADDRESS] [f/food] [dt/DATETIME]` +
 e.g. `/order find n/James Jake`
 * *Delete order* : `/order delete INDEX` +
 e.g. `/order delete 3`
 * *Clear order* : `/order clear`
 * *Add delivery man* : `/deliveryman add n/NAME` +
 e.g. `/deliveryman add n/James Ho`
 * *Listing delivery men* : `/deliveryman list`
-* *Edit delivery man* (Coming in v2.0) : `/deliveryman edit INDEX [n/NAME] [p/PHONE_NUMBER] [a/ADDRESS]` +
+* *Edit delivery man* (Coming in v2.0) : `/deliveryman edit INDEX n/NAME` *[Coming in v2.0]* +
```

Comment:
The `[Coming in v2.0]` at the end can be deleted since there's already one next to *Edit delivery man*